style.css

```diff
@@ -0,0 +1,7 @@
+body {background-color: moccasin;}
```

Comment:
In CSS, we format the code a little differently. This still works, but it is not easy to read.
Please reformat this file so that each style of CSS is on its own line.
Ex:
h1 {
  color: royalblue;
  margin: 20px; 
  text-align: center;
}